example/lib/icons.dart

```diff
@@ -1,1625 +1,1618 @@
+
+import 'package:flutter/material.dart';
```

Comment:
Not sure why we need `flutter/material.dart` here

Reply:
This may not actually be needed anymore- it may be a remnant of another approach that I tried.  I'll double check.